ml_frameworks_bot/keras/eval.py

```diff
@@ -0,0 +1,184 @@
+import weave
+from weave import Dataset
+
+# Initialize Weave
+weave.init(project_name="ml-colabs/ml-frameworks-bot")
+
+# Create a dataset
+eval_dataset = Dataset(
```

Comment:
You should simply publish the dataset and use its `ref` for evaluation.
Like this
```python
dataset = weave.Dataset(name="awesome-ds", rows=[...])
weave.publish(dataset)
```